analyses/molecular-subtyping-embryonal/README.md

```diff
@@ -1,10 +1,80 @@
+- [Usage](#usage)
+- [Folder Content](#folder-content)
+  - [Define samples for subsetting](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/01-samples-to-subset.nb.html)
+  - [Generate subset files](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-embryonal/02-generate-subset-files.R)
+  - [Clean C19MC miRNA cluster data](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/03-clean-c19mc-data.nb.html)
+  - [Construct summary tables](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/04-table-prep.nb.html)
+- [Folder Structure](#folder-structure)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Usage
+
 When re-running this module, you may want to regenerate the subset files using the most recent data release.
-Files will be regenerated using the symlinked files in `data` by default when running:
+Files will be regenerated using the symlinked files in `data` by default when running from the command line as follows:
 
-```sh
+```
 bash run-embryonal-subtyping.sh
 ```
+
+
+## Folder Content
+
+[`01-samples-to-subset.Rmd`](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/01-samples-to-subset.nb.html) is a notebook written to identify samples to include in subset files for the purpose of molecularly subtyping non-MB and non-ATRT embryonal tumors.
+The samples are identified using the following criteria:
+
+1. An RNA-seq biospecimen sample includes a _TTYH1_ fusion (5' partner).
+2. Any sample with "Embryonal tumor" in the `broad_histology` column of the metadata `pbta-histologies.tsv` that is not labeled "Medulloblastoma" or "Atypical Teratoid Rhabdoid Tumor (ATRT)" in `disease_type_old` or `disease_type_new` columns [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/251#issuecomment-568220913).
+The output of this notebook is a tsv file containing the biospeciemen IDs identified based on the above criteria (stored in the `results` directory of this module.
+
+[`02-generate-subset-files.R`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-embryonal/02-generate-subset-files.R) is a script written to subset the files required for the subtyping of non-MB and non-ATRT embryonal tumors.
+The output of this script includes the subsets of the structural variant, poly-A RNA-seq, and stranded RNA-seq data files (stored in the `subset-files` directory of this module).
+
+[`03-clean-c19mc-data.Rmd`](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/03-clean-c19mc-data.nb.html) is a notebook written to clean copy number data related to C19MC amplifications in non-MB, non-ATRT embryonal tumors.
+Specifically, the goal of this notebook is to identify embryonal tumors with multilayered rosettes (ETMR), C19MC-altered tumors.
```

Comment:
What is the notebook doing that helps us to identify the ETMR, C19MC-altered tumors?

Reply:
I updated this section, is the description now suffice or would you prefer that I add another layer of detail?

Reply:
A couple minor distinctions - we look at segments with positive `seg.mean` on chromosome 19 and I think the visualization we do is important to note. That's kind of the only way we look at the _width_ of the amplification. The rationale for this notebook is that there was some disagreement about what the genomic location of C19MC was.

Reply:
Ahh, okay I will make note of that.